yarn.lock

```diff
@@ -1590,6 +1590,11 @@
   resolved "https://registry.yarnpkg.com/@lumino/algorithm/-/algorithm-1.2.0.tgz#74e20e2ccf8c1452dde3dbc7887d9c01d0584031"
   integrity sha512-QZpTDiqY53HmoPVDjsqJ3H6k8uhKIrbrOLV15wEkQMJJW38SzMnm+GxCXHRJ6J5ivL2L7lzyZ5UIVTUirscMMA==
 
+"@lumino/algorithm@^1.2.1":
```

Comment:
Can you reset this yarn file back to have no changes, then `jlpm install` again to update it? We're picking up some duplicates here. Alternatively, you could just hand-delete all of the `@lumino` entries and run `jlpm install` - that should take care of the duplicates.

Reply:
Or `jlpm deduplicate`?

Reply:
I think I tried `jlpm deduplicate`.  Ultimately, it was simpler to just remove all the @lumino entries and install again.